helm/flux-app/templates/source.yaml

```diff
@@ -22,6 +34,18 @@ type: Opaque
 data:
   username: "{{ .credentials.username | b64enc }}"
   password: "{{ .credentials.password | b64enc }}"
+  token: "{{ .credentials.password | b64enc }}"
```

Comment:
Are these supposed to point to the same key?

Reply:
yes